### ISSUE TITLE
[Question] Is there a way to load encodings on demand?

### ISSUE THREAD

**Issue by eekboom** [ashtuchkin/iconv-lite#291]
After adding iconv-lite to my small preact app, the minified js bundly grew by about 300KB.

Is there a way to selectively bundle a few encodings (utf-8, iso-8859-*) and fetch the others when needed?
I guess there isn't because the API to encode/decode is synchronous.

In that case it would be a very welcome feature.

**Comment by ashtuchkin** [comment]
That's correct. I guess a good way to do that would be to split encodings into different npm modules and give users the option to only depend on the one they need. I'll take that into account when planning new version. 

Thank you!
